build.gradle.kts

```diff
@@ -41,7 +41,7 @@ dependencies {
 kotlin {
   jvmToolchain(21)
   compilerOptions {
-    freeCompilerArgs.add("-Xwhen-guards")
+    freeCompilerArgs.addAll("-Xjvm-default=all", "-Xwhen-guards")
```

Comment:
need the jvm default so that the interfaces get generated with a default implementation and can bodies can then be specified